parallel-install/pkg/deployment/overrideinterceptors.go

```diff
@@ -100,10 +139,18 @@ func (i *DomainNameOverrideInterceptor) getDomainName() (domainName string, err
 		return "", err
 	}
 	if domainName != "" {
+		err = patchCoreDNS(i.kubeClient, `(.*)\.local\.kyma\.devaaaa`, i.log)
```

Comment:
We are patching anyway few lines below with a different argument. Perhaps we can simplify and this invocation can be avoided.

Reply:
I think it will just move code repetition to another place or create more code (another if/switch based on domain name)